jupyterhub/templates/NOTES.txt

```diff
@@ -1,54 +1,90 @@
-Thank you for installing JupyterHub!
+{{- $proxy_service := include "jupyterhub.proxy-public.fullname" . -}}
```

Comment:
I see that we use snake case for local go template variables elsewhere in z2jh too! I thought we were doing camelCase but I was wrong!